core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -0,0 +1,105 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the top navigation bar.
+ */
+
+oppia.directive('topNavigationBar', [function() {
+  return {
+    restrict: 'E',
+    scope: {},
+    templateUrl: 'components/topNavigationBar',
+    controller: [
+      '$scope', '$http', '$window', '$timeout', 'UrlInterpolationService',
+      'SidebarStatusService', 'LABEL_FOR_CLEARING_FOCUS',
+      'siteAnalyticsService', 'windowDimensionsService',
+      function(
+          $scope, $http, $window, $timeout, UrlInterpolationService,
+          SidebarStatusService, LABEL_FOR_CLEARING_FOCUS,
+          siteAnalyticsService, windowDimensionsService) {
+        $scope.NAV_MODE = GLOBALS.NAV_MODE;
+        $scope.LABEL_FOR_CLEARING_FOCUS = LABEL_FOR_CLEARING_FOCUS;
+        $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+
+        $scope.onLoginButtonClicked = function(loginUrl) {
+          siteAnalyticsService.registerStartLoginEvent('loginButton');
+          $timeout(function() {
+            $window.location = loginUrl;
+          }, 150);
+          return false;
```

Comment:
Why return anything if it will always be false? Does this need to return a value?